package.json

```diff
@@ -26,7 +26,7 @@
     "prepublish": "gulp transpile",
     "test": "npm run lint && ava",
     "lint": "xo",
-    "format": "prettier --single-quote --no-semi --write *.js {src,test}/*.js",
+    "format": "prettier --single-quote --no-semi --write *.js {src,test}/*.js test/fixtures/*-prettier.js",
```

Comment:
The thinking was that if a prettier 1.6 comes out we want to ensure it doesn't break the babel plugin parsing. A bit of an edge case i guess